src/main/java/frc/robot/arm/ArmIO.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ArmIO {
+  @AutoLog
+  public static class ArmIOInputs {
+    public double angularVelocityRotsPerSec = 0.0;
+    public Rotation2d position = new Rotation2d();
+    public double supplyCurrentAmps = 0.0;
+    public double appliedVoltage = 0.0;
+    public double statorCurrentAmps = 0.0;
+  }
+
+  void updateInputs(ArmIOInputs inputs);
+
+  void setMotorVoltage(double voltage);
+
+  void setMotorPosition(Rotation2d targetPosition);
+
+  void setRollerVoltage(double voltage);
```

Comment:
```suggestion
  public void updateInputs(ArmIOInputs inputs);

  public void setMotorVoltage(double voltage);

  public void setMotorPosition(Rotation2d targetPosition);

  public void setRollerVoltage(double voltage);
```
like, these are all *implicitly* public, but i'd feel a lot better if they were also marked as such